go.work

```diff
@@ -1,6 +1,5 @@
 go 1.24
 
-use (
-	.
-	api
-)
+use .
+
+replace jester.noble.xyz/api v1.0.0 => ./api
```

Comment:
it's not immediately obvious to me why we need this replace statement now

Reply:
Without this, whenever I run `go work sync` the Go version in the api gets updated. 
Also, adding this `replace` removes the unneeded indirect imports in the api module that are tied to `tools`.

Reply:
I actually do think removing `go.work` entirely is the best option. Lmk if you disagree!
[`fdad901` (#11)](https://github.com/noble-assets/jester/pull/11/commits/fdad9012bf43b9245465b315bd39dfbbe923d350)

Reply:
Yes definitely a fan of not checking in `go.work` files. 

`go.work` is meant to be used locally to point to dependencies on your machine vs. remote copies of those same dependencies. Typically if you want to check `go.work` into a remote repository you will add it as `go.work.example` so others can use it as an example for their own local workspace.